qemu/tests/single_driver_install.py

```diff
@@ -172,6 +174,15 @@ def run(test, params, env):
         if not utils_misc.wait_for(lambda: not session.cmd_status(chk_cmd),
                                    600, 60, 10):
             test.fail("Failed to install driver '%s'" % driver_name)
+        if device_name[0:30] == "Red Hat VirtIO Ethernet Adapte":
```

Comment:
I think there is a typo on the device name.

Reply:
@vivianQizhu, That is the name of device_name. It was in effect in our recent test log.

Reply:
My bad. I see my mistake. I will fix it.

Reply:
I have fixed it. Could you help review it again? Thanks a lot!